Lib/test/test_abc.py

```diff
@@ -392,6 +392,13 @@ class MyInt(int):
             self.assertIsInstance(42, A)
             self.assertIsInstance(42, (A,))
 
+        def test_issubclass(self):
```

Comment:
I would rather make the test name more specific, or add few more corner cases here.

Reply:
+1